src/main/scala/akka/minion/HttpServer.scala

```diff
@@ -93,7 +93,7 @@ class HttpServer(
       }
 
   override def preStart(): Unit = {
-    bindingFuture = Http(context.system).bindAndHandle(route, "0.0.0.0", settings.httpPort)
+    bindingFuture = Http(context.system).newServerAt("0.0.0.0", settings.httpPort).bindFlow(route)
```

Comment:
The Scalafix rule almost did it, but route is not a Flow.

Reply:
The implicit conversion didn't apply as it lacked the implicit `ClassicActorSystemProvider`. I wonder if we can improve the compilation error with annotations in Akka HTTP.

Reply:
Interesting, so here a Materializer used to be enough but now it isn't.